packages/ethtool/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "ethtool"
+version = "0.1.0"
+edition = "2018"
```

to sync up with #2835:
```suggestion
edition = "2021"
```

---

There are merge conflicts in #2835 now, so @yeazelm if you prefer to merge this as-is, I will pick up the change and bump it in the other PR.